config/schedule.yml

```diff
@@ -150,3 +150,7 @@ notify_about_published_articles:
   description: "Sends notifications about the new published articles (runs every minute)"
   cron: "*/5 * * * *"
   class: "Articles::PublishWorker"
+display_ad_event_rollup:
+  description: "Compact rows in the display_ad_events table"
```

Comment:
nit: the unspoken thing we've been doing is to include, in parenthesis at the end, the frequency of the said cron job.